.github/workflows/mango-ui-deploy-npm.yml

```diff
@@ -13,10 +13,12 @@ jobs:
     - name: Set up Node.js
       uses: actions/setup-node@v4
       with:
-        node-version: '14.x'
+        node-version: '21.x'
```

Comment:
@HelenaCA this was the problem. Older version of node in the worklfow was causing the build to silently fail and not create a dist folder so the upload was just the rest of the files